moz_security/sandboxes/heka/analysis/moz_security_sshd_login_monitor.lua

```diff
@@ -5,24 +5,47 @@
 --[[
 #  Mozilla Security SSHD Login Monitor
 
-Sends a message to Pagerduty anytime there is a successful Bastion sshd login.
-Pagerduty will then alert if it is out of policy (hours) for that user.
+Match SSH login events and generate email alerts.
+
+By default, alerts always go to default_email.
+
+If user_email is specified in the configuration, alerts will also be sent to this
+address. User email should be a string containing a %s format specifier which is
+replaced with the username of the account which logged in.
+
+The acceptable_message_delay parameter indicates an age in seconds. If a new event
```

Comment:
Newer would also be bad

Reply:
Yeah that'd be an odd scenario, likely different root cause; I don't know if we'd want to handle the case of a newer event any differently that the normal case?

Reply:
I agree on a different root cause but they are both an operational issues.  I would just compare the abs of the delta and alert (with the signed value) if the time appears to be out of wack.